### PR TITLE
chore: update open-next version in Nextjs component 

### PR DESCRIPTION
**Pull request by sommeeeer**
This PR bumps the default OpenNext version to be the latest which is `3.1.4`. It also changes the build command to address that OpenNext => `3.1.13` needs to use the new package called: `@opennextjs/aws`. 

If its `3.1.13` or lower it will use the old `open-next` package.

### CODE REVIEW

**Review by conico974 — APPROVED** [review]
LGTM

File: platform/src/util/compare-semver.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const [major1, minor1, patch1] = v1.split(".").map(Number);
+  const [major2, minor2, patch2] = v2.split(".").map(Number);
+
+  if (major1 !== major2) return major1 - major2;
+  if (minor1 !== minor2) return minor1 - minor2;
+  return patch1 - patch2;
+}
+
+export function getOpenNextPackage(openNextVersion: string): string {
+  return compareSemver(openNextVersion, "3.1.13") <= 0 ? "open-next" : "@opennextjs/aws";
```
Comment (socsieng) [inline platform/src/util/compare-semver.ts:18]: Is this a typo? Should it be `3.1.3`?

Comment (conico974) [reply platform/src/util/compare-semver.ts:18]: Ho yeah definitely a typo @sommeeeer @thdxr

Comment (sommeeeer) [reply platform/src/util/compare-semver.ts:18]: > Is this a typo? Should it be `3.1.3`?

nice spotted! making a new PR to address this now.

**Pull request merged**